internal/nodes/avi_model_l4_translator.go

```diff
@@ -519,7 +519,7 @@ func PopulateServers(poolNode *AviPoolNode, ns string, serviceName string, ingre
 	if len(svcObj.Spec.IPFamilies) == 2 {
 		v4Family = true
 		v6Family = true
-	} else if svcObj.Spec.IPFamilies[0] == "IPv6" {
+	} else if len(svcObj.Spec.IPFamilies) > 0 && svcObj.Spec.IPFamilies[0] == "IPv6" {
```

Comment:
During ingestion of service, we check type of service and log a message saying `ExternalName service is not supported` and return from there, will help in avoid going through all remaining logic.

Reply:
fixed